includes/class-sensei-admin.php

```diff
@@ -348,9 +366,7 @@ public function register_scripts( $hook ) {
 		Sensei()->assets->enqueue( 'sensei-event-logging', 'js/admin/event-logging.js', [ 'jquery' ], true );
 
 		// Sensei custom navigation.
-		if ( $screen && ( in_array( $screen->id, [ 'edit-course', 'edit-course-category' ], true ) ) ) {
-			Sensei()->assets->enqueue( 'sensei-admin-custom-navigation', 'js/admin/custom-navigation.js', [], true );
-		}
+		Sensei()->assets->enqueue( 'sensei-admin-custom-navigation', 'js/admin/custom-navigation.js', [], true );
```

Comment:
Previously @donnapep asked to enqueue these scripts only when they are needed: https://github.com/Automattic/sensei/pull/4672#discussion_r791904559

In my new PR with Modules navigation I extended this check this way: https://github.com/Automattic/sensei/pull/4693/files#diff-a700db88caca1edaa7ce24fc214d9344458fb9a1a636e4f8fa6bfb9eaa9a93dfR372-R375

Reply:
I will then return the old if - logic and add it to the array, and when you merge you branch I can move it to your list, not to cause conflicts